api/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -1044,15 +1044,33 @@ public void updateOobChannelAddresses(ManagedChannel channel, EquivalentAddressG
      * {@link ManagedChannelBuilder#forTarget} for the format of a target string.
      *
      * <p>The target string will be resolved by a {@link NameResolver} created according to the
-     * target string.  The out-of-band channel doesn't have load-balancing.  If multiple addresses
-     * are resolved for the target, the first working address will be used.
+     * target string.  If multiple addresses are resolved for the target, the first working address
+     * will be used.
```

Comment:
I think the last sentence is not needed as well. Basically it will use the balancer determined by the service config from the NameResolver or the default config.

Reply:
yes, removed.